api/HTMLInputElement.json

```
@@ -1863,6 +1863,56 @@
           }
         }
       },
+      "selectionchange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/selectionchange_event",
+          "spec_url": "https://w3c.github.io/selection-api/#selectionchange-event",
+          "description": "<code>selectionchange</code> event",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "92"
+            },
+            "firefox_android": {
+              "version_added": "92"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
```

Sorry, I missed this last time. This should be true, since it's a single implementation (for now).

```suggestion
            "experimental": true,
```